src/pudl/helpers.py

```diff
@@ -1918,3 +1923,84 @@ def diff_wide_tables(
     return TableDiff(
         deleted=deleted, added=added, changed=changed, old_df=old, new_df=new
     )
+
+
+def create_datasette_metadata_yaml() -> str:
```

Comment:
I made the datasette metadata generation a helper function so parts of the code base that generate datasette metadata are all working with the same logic.

Reply:
Great, I think bringing as much (maybe all...) of our devtools / scripty stuff under the main package hierarchy is probably a good idea. It's starting to grow over there.